pom.xml

```diff
@@ -1,17 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?><!--
-  ~ Copyright 2018-2019 adorsys GmbH & Co KG
+<!--
```

Comment:
Error:      Non-parseable POM /home/runner/work/open-banking-gateway/open-banking-gateway/pom.xml: in comment after two dashes (--) next character must be > not \n (position: START_DOCUMENT seen <?xml version="1.0" encoding="UTF-8"?><!--\n<!--\n... @3:1)  @ line 3, column 1 -> [Help 2]

Reply:
Done.